src/components/NewsCard/index.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import {
+    Box,
+    Text,
+    VStack,
+    useColorModeValue,
+    Flex,
+    Spacer,
+    Divider
+} from '@chakra-ui/react';
+import { CloseIcon, DragHandleIcon } from "@chakra-ui/icons"
+
+import NewsArticle from "../NewsArticle";
+
+const NewsCard = ({source, price, buttonText, details, ...otherProps}) => {
+    const boxColor = useColorModeValue("brand.100", "brand.700");
+
+    return (
+        <Box mx={3} mt={5} px={4} py={4} borderRadius="lg" shadow="md" bg={boxColor} {...otherProps}>
+            <Flex>
+                <DragHandleIcon />
+                <Spacer />
+                <CloseIcon />
+            </Flex>
+            <VStack>
```

Comment:
I think the aligned to the left has to be specified on VStack 
It aligns to the center for smaller title 
![image](https://user-images.githubusercontent.com/31973210/121766173-fecbaa00-cb04-11eb-82fe-f5a8b0e4a66e.png)

Reply:
I'm unable to reproduce what your screenshot shows, and I tested various window widths/heights. How did you get it like that?

I'd already set align=left in the NewsArticle VStack. I will add it to the NewsCard VStack too.
Please let me know if you're still seeing the issue